src/components/icon-extention-with-title/IconExtentionWithTitle.styles.ts

```diff
@@ -0,0 +1,31 @@
+import { TypographyVariantEnum } from '~/types'
+import palette from '~/styles/app-theme/app.pallete'
+
+export const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  iconBox: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '40px',
+    height: '40px',
+    backgroundColor: palette.basic.turquoise,
+    borderRadius: '5px',
+    marginRight: '20px',
+    color: palette.basic.white,
```

Comment:
Don't need to import palette, you can just write as 
`backgroundColor: 'basic.turquoise'` 
`color: 'basic.white'`

Reply:
fixed